libraries/AP_Scripting/lua_bindings.cpp

```diff
@@ -532,3 +535,65 @@ int lua_removefile(lua_State *L) {
   const char *filename = luaL_checkstring(L, 1);
   return luaL_fileresult(L, remove(filename) == 0, filename);
 }
+
+
+// local cmd, data = crsf_pop() -- get byte array from crsf or nil if no data received
+int lua_crsf_pop(lua_State *L) {
+#if HAL_CRSF_TELEM_ENABLED
```

Comment:
You can just def out the whole method.

Reply:
We might need to mess with the generator to get it  to cope.

Reply:
Deffing out the whole method was also my first thought, but have no idea how the generator reacts.

Reply:
doing a sub table should allow you to add a `depends` to the table. Again see `CAN`